File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Tpp/Models/Prescriptions/Medication.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Tpp.Models
{
    [Serializable]
    public class Medication
    {
        [XmlAttribute("drugId")]
        public string DrugId { get; set; }
        
        [XmlAttribute("type")]
        public string Type { get; set; }
        
        [XmlAttribute("drug")]
        public string Drug { get; set; }
        
        [XmlAttribute("details")]
        public string Details { get; set; }
       
        [XmlAttribute("requestable")]
        public string Requestable { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/Medication.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "Medication")
@XmlAccessorType(XmlAccessType.FIELD)
data class Medication(
        @XmlAttribute var drugId: String = "default drugId",
        @XmlAttribute var type: String = "default type",
        @XmlAttribute var drug: String = "default drug",
        @XmlAttribute var details: String = "default details",
        @XmlAttribute var requestable: String = "default requestable"
)
